apps/studio/components/interfaces/DataWarehouse/TestCollectionDialog.tsx

```diff
@@ -0,0 +1,153 @@
+  const selectedCollectionName = collections.find((col) => col.token === selectedCollection)?.name
+
+  return (
+    <Dialog>
+      <Tooltip_Shadcn_>
+        <TooltipTrigger_Shadcn_ asChild>
+          <DialogTrigger asChild>
+            <Button disabled={accessTokens.length === 0} type="outline">
+              Connect
+            </Button>
+          </DialogTrigger>
+        </TooltipTrigger_Shadcn_>
+        {accessTokens.length === 0 && (
+          <TooltipContent_Shadcn_>
+            Create an access token to connect to your collection
+          </TooltipContent_Shadcn_>
+        )}
+      </Tooltip_Shadcn_>
+
+      <DialogContent>
+        <DialogHeader>
+          <DialogTitle>Send events to this collection</DialogTitle>
+          <DialogDescription>
+            Use the following curl command to send events to this collection
+          </DialogDescription>
+        </DialogHeader>
+        <DialogSectionSeparator />
+        <DialogSection className="flex flex-col gap-4 overflow-auto">
+          <div className="flex gap-2 *:flex-1">
+            <FormItemLayout label="Collection" isReactForm={false}>
+              <Select value={selectedCollection} onValueChange={setSelectedCollection}>
+                <SelectTrigger>
+                  <span className="text-ellipsis">{selectedCollectionName || 'Collection'}</span>
+                </SelectTrigger>
+                <SelectContent>
+                  {collections.map((col) => (
+                    <SelectItem key={col.id + '-collection'} value={col.token}>
+                      {col.name || 'No name'}
+                    </SelectItem>
+                  ))}
+                </SelectContent>
+              </Select>
+            </FormItemLayout>
+            <FormItemLayout label="Token" isReactForm={false}>
+              <Select value={testAccessToken} onValueChange={setTestAccessToken}>
+                <SelectTrigger>
+                  <span className="text-ellipsis">
+                    {selectedAccessToken?.description || 'Access token'}
+                  </span>
+                </SelectTrigger>
+                <SelectContent>
+                  {accessTokens?.map((token: any) => (
+                    <SelectItem key={token.id + '-token'} value={token.token}>
+                      {token.description || 'No description'}
+                    </SelectItem>
+                  ))}
+                </SelectContent>
+              </Select>
+            </FormItemLayout>
+          </div>
+
+          <FormItemLayout label="Ingest URL" isReactForm={false}>
+            <Input
+              className="font-mono tracking-tighter"
+              value={BASE_WAREHOUSE_URL}
+              readOnly
+              copy
+            />
+          </FormItemLayout>
+
+          <div>
+            <CodeBlock className="p-1 language-bash prose" language="bash">
+              {`curl -X "POST" "${BASE_WAREHOUSE_URL}?source=${selectedCollection}" \\
```

Comment:
there is also the alternate way to send events, which is throguh the `?source_name=` query param, which is more readable.
we can stick  with the uuid example for now